spec/models/user_spec.rb

```diff
@@ -134,6 +134,14 @@ def is_shop_or_fehb_market_enabled?
       end
     end
 
+    context 'when password has white spaces' do
+      let(:params){valid_params.deep_merge!({password: "Test@1234 "})}
```

Comment:
I believe we have to allow "Test@1234 " by squishing input to remove whitespaces around it but, should be blocking "Test @1234". what do you think?
(I posted this on story to get answer from business)

Reply:
As per the discussion, we should not be allowing the user to enter white spaces. in password